code/espurna/debug.ino

```diff
@@ -60,6 +60,14 @@ void _debugSend(char * message) {
         _telnetWrite(message, strlen(message));
     #endif
 
+    #if DEBUG_WEB_SUPPORT
+        #if DEBUG_ADD_TIMESTAMP
+            wsSend_P(PSTR("{\"weblog\": \"%s %s\"}"), timestamp, message);
```

Comment:
This can be just `wsSend(message)` and timestamp can always be in the client via `Date.toLocaleTimeString()`

Reply:
In theory, yes (and it would look better), but this is consistent with Telnet and Serial messages. 
I'm happy to add it as optional field before the actual text received from device - would this be a good idea?